player/command.c

```diff
@@ -7139,6 +7143,7 @@ const struct mp_cmd_def mp_cmds[] = {
                 .flags = MP_CMD_OPT_ARG},
             {"title", OPT_STRING(v.s), .flags = MP_CMD_OPT_ARG},
             {"lang", OPT_STRING(v.s), .flags = MP_CMD_OPT_ARG},
+            {"hearing-impaired", OPT_BOOL(v.b), .flags = MP_CMD_OPT_ARG},
```

Comment:
My only slight hesitation with adding this argument to the command is that it's not very extensible. What if someone wants to make a track the default, forced, etc? I know we don't offer this level of control currently but if we are going to add stuff to this, it needs to be more than just adding more arguments to this command imo. I dunno maybe something like key/value lists?

Reply:
This can be `OPT_FLAGS` to make this extensible. Note that we already have `{"albumart", OPT_BOOL(v.b), .flags = MP_CMD_OPT_ARG},` in `video-add`, so this is kinda the same here.

Reply:
changed now